lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -759,6 +761,7 @@ private void createLabelPool() {
       // if more thumbs are added.
       TooltipDrawable tooltipDrawable =
           TooltipDrawable.createFromAttributes(getContext(), null, 0, labelStyle);
+      tooltipDrawable.setLayoutMargin(0);
```

Comment:
would you like to create a separate PR to fix the label breaking issue (or change this one)? for that we can change the layout margin to 2dp instead of 0dp (so it doesn't directly touch the screen edges but it also doesn't break the label), and the change can be done in the tooltip style [here](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/tooltip/res/values/styles.xml#L37)!